src/services/SubmissionService.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+export class SubmissionService {
+
+    constructor() {
+        this.axios = axios.create({
+            baseURL: process.env.REACT_APP_PROBLEM_SVC_BASE_URL,
+            timeout: 10000
+        });
+    }
+
+    async submit(problem) {
```

Comment:
Should be renamed to `rejudge`

and don't implement this method at this moment, the issue created for this #76